src/evo/deterministicmns.cpp

```diff
@@ -354,14 +354,23 @@ CSimplifiedMNListDiff CDeterministicMNList::BuildSimplifiedDiff(const CDetermini
     CSimplifiedMNListDiff diffRet;
     diffRet.baseBlockHash = blockHash;
     diffRet.blockHash = to.blockHash;
+    diffRet.nVersion = llmq::utils::IsBasicBLSSchemeActive(::ChainActive().Tip()) ? CSimplifiedMNListDiff::LEGACY_BLS_VERSION : CSimplifiedMNListDiff::BASIC_BLS_VERSION;
```

Comment:
this is setting `nVersion` like this `nVersion = isBasicActive ? Legacy : Basic` so it will inform the SPV clients the incorrect scheme.

Reply:
@HashEngineering Yup will fix that